server/db/dbUtils.js

```diff
@@ -29,6 +29,13 @@ export const insertAllQuery = (schema, table, cols, itemsValues) => {
   return pgp.helpers.insert(valuesIndexedByCol, columnSet)
 }
 
+export const insertAllQueryBatch = (schema, table, cols, itemsValues) => {
```

Comment:
why not to use the one above ?

Reply:
because the one above indexed the values by col, I that the ideal thing is use this ( the new) into the old one after index the values, I will do that!